packages/document-model-libs/editors/chat-room-editor/chat-room-editor.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import Editor from "./editor";
+import { createDocumentStory } from "document-model-libs/utils";
+
+import * as ChatRoomModule from "../../document-models/chat-room";
+
+const { meta, CreateDocumentStory: ChatRoom } = createDocumentStory(
+  Editor,
+  ChatRoomModule.reducer,
+  ChatRoomModule.utils.createDocument(),
+);
+export { ChatRoom };
+
+export default { ...meta, title: "Chat Room Editor" };
```

Comment:
When fixing the typescript build issues, I found that we were not actually type checking these story files correctly. I think it is still possible to export the default like this, but there is a config somewhere that would need to be done that I can't seem to find.

For now all that needs to change is adding ` as Meta<typeof Editor>;` to the export default, so like 

`export default { ...meta, title: "Chat Room Editor" } as Meta<typeof Editor>;` 

this lets typescript know that the default export which is not named is in fact using these storybook nested types.